### ISSUE TITLE
service details doesn't update with the service overview

### ISSUE THREAD

**Issue by matti** [kontena/kontena-cloud-issues#8]
![kontena_cloud](https://user-images.githubusercontent.com/3893/27073426-77c89af6-502c-11e7-895e-df3b218aee58.png)

**Comment by nevalla** [comment]
Yes this is known issue. Kontena Master does not send (yet) service instance information in real time so you have to re-open service panel to get that information.
